app/lib/hosting_environment.rb

```diff
@@ -14,7 +14,7 @@ def self.local_development?
   end
 
   def self.dev?
-    environment_name == "dev"
+    environment_name == "dev" || environment_name == "paas_dev"
```

Comment:
I;m a bit confused about this, I thought we agreed not to mention platform which the app is deployed too. I really dont think our app should worry about whether its in paas or aws. it should only care about whether its dev/staging/prod/ur

Reply:
We agreed that we shouldn't put anything platform-related in the variable name, but Dan said we could put platform information in the variable values if necessary (in this case adding the `paas_` prefix enables Laurence's script to differentiate between the different platforms)